Assets/Scripts/StandardSamples/UI/Common/UINativeLibVersion.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PlayEveryWare.EpicOnlineServices.Samples
+{
+    using UnityEngine;
+    using UnityEngine.UI;
+
+    public class UINativeLibVersion : MonoBehaviour
+    {
+        private void Start()
+        {
+            var textComp = GetComponent<Text>();
+            textComp.text = $"v-{EOSPackageInfo.NativeLibVersion}";
```

Comment:
Minor: Add null check
```suggestion
        if (textComp != null)
        {
            textComp.text = $"v-{EOSPackageInfo.NativeLibVersion}";
        }
```

Reply:
Added null check